docs/generateClientCodeChallenge.md

```diff
@@ -70,6 +70,8 @@ With the application still running, you can run the [test code](https://github.i
 python ./test/test_apis.py
 ```
 
+*Note: Requires Python 2.7 and the app should be restarted before running tests to clear the cache.*
```

Comment:
Updated on Influitive, however, I've moved this up into the other dependency note about urllib3, certifi and six as it's best dealt with before running any python code:

"...may include `urllib3`, `certifi` and `six`. The tools provided in the repository are intended for use with Python 2.7."

Reply:
Good idea. Updated.